README.md

```
@@ -32,6 +33,31 @@ ex) `week2/programmers: 폰켓몬`
 |신고 결과 받기|⭐⭐|https://programmers.co.kr/learn/courses/30/lessons/92334|
 |메뉴 리뉴얼|⭐⭐⭐|https://programmers.co.kr/learn/courses/30/lessons/72411|
 
+<br/>
+
+## week2 (04/07-04/14)
+####  ✅ 필수 문제
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)